emails/NewRequest.tsx

```diff
@@ -0,0 +1,67 @@
+import { PrivateCloudCreateRequestBody } from '@/schema';
+import * as React from 'react';
+import Header from './components/Header';
+import ProductDetails from './components/ProductDetails';
+import { Body, Button, Heading, Html, Img, Text } from '@react-email/components';
+import { Tailwind } from '@react-email/tailwind';
+import { sampleFormData } from './components/Params';
+import NamespaceDetails from './components/NamespaceDetails';
+import Closing from './components/Closing';
+import { TailwindConfig } from './components/TailwindConfig';
+const defaultTheme = require('tailwindcss/defaultTheme');
+
+interface EmailProp {
+  formData: PrivateCloudCreateRequestBody;
+}
+
+export const NewRequest = ({ formData }: EmailProp) => {
+  if (!formData) {
+    formData = sampleFormData;
```

Comment:
It seems that the fallback to `sampleFormData` is intended to support `local dev or dev` environments. However, I believe it would be better to throw an error or return an empty string if no data is passed into the template.